scripts/bundled_installer

```diff
@@ -31,14 +34,21 @@ function exit_if_return_code_is_non_zero() {
     fi
 }
 
+function echo_recommendation_message() {
```

Comment:
You should be able to get rid of this function just as you did [here](https://github.com/aws/aws-elastic-beanstalk-cli-setup/pull/54#pullrequestreview-282192972)